src/components/Tag.js

```diff
@@ -43,8 +44,8 @@ const OuterBorder = styled(Centered)`
   z-index: 2;
 `;
 
-const Text = styled(TextElement).attrs(({ theme: { colors } }) => ({
-  color: colors.whiteLabel,
+const Text = styled(TextElement).attrs(({ theme: { colors }, max }) => ({
+  color: max ? lightModeThemeColors.grey : colors.whiteLabel,
```

Comment:
I use `lightModeThemeColors`, because the layout of this screen is not changing with theme